### ISSUE TITLE
[v4.0.0-beta.14] Lines not showing up on ion-item

### ISSUE THREAD

**Issue by jordanpurinton** [ionic-team/ionic#16164]
# Bug Report

**Ionic Info**
```
Ionic:

   ionic (Ionic CLI)             : 4.2.1 (path\npm\node_modules\ionic)
   Ionic Framework               : @ionic/angular 4.0.0-beta.14
   @angular-devkit/build-angular : 0.8.3
   @angular-devkit/schematics    : 0.8.3
   @angular/cli                  : 6.2.5
   @ionic/angular-toolkit        : not installed

Cordova:

   cordova (Cordova CLI) : 8.1.1 (cordova-lib@8.1.0)
   Cordova Platforms     : android 7.1.1
   Cordova Plugins       : cordova-plugin-ionic-keyboard 2.1.3, cordova-plugin-ionic-webview 2.1.4, (and 10 other plugins)

System:

   NodeJS : v8.12.0 (C:\Program Files\nodejs\node.exe)
   npm    : 6.4.1
   OS     : Windows 10
```

**Describe the Bug**
After upgrading to beta 14, none of my `ion-item` components have lines underneath them (using lines="full").

**Expected Behavior**
For the lines to show up.

**Steps to Reproduce**
Upgrade to beta 14
See absence of lines on ion-item lines="full"

**Comment by ireade** [comment]
Also see https://github.com/ionic-team/ionic/issues/16163

**Comment by manucorporat** [comment]
Should be fixed in 0.15.0-0, it was related with a bug in the CSS minifier:
https://github.com/cssnano/cssnano/issues/652
